doc/radosgw/s3select.rst

```diff
@@ -0,0 +1,265 @@
+         +-------+    +-------+                                   +-----+   +-----+
+         |payment|    |  0.3  |                                   | EQ  |   | LT  |
+         +-------+    +-------+                                +--+-----+   +-----+--+
+                                                               |        |   |        |
+                                                               |        |   |        |
+                                                               V        V   V        V
+                                                          +-------+ +----+ +-----+ +-----+
+                                                          | region| |east| |age  | | 30  |
+                                                          +-------+ +----+ +-----+ +-----+
+
+
+
+Features Support
+----------------
+
+.. _feature-table:
+
+The following table describes the support for s3-select functionalities:
+
++---------------------------------+-----------------+-----------------------------------------------------------------------+
+| Feature                         | Detailed        | Example                                                               |
++=================================+=================+=======================================================================+
+| Arithmetic operators            | ^ * / + - ( )   | select (int(_1)+int(_2))*int(_9) from stdin;                          |
++---------------------------------+-----------------+-----------------------------------------------------------------------+
+|                                 |                 | select ((1+2)*3.14) ^ 2 from stdin;                                   |
++---------------------------------+-----------------+-----------------------------------------------------------------------+
+| Compare operators               | > < >= <= == != | select _1,_2 from stdin where (int(1)+int(_3))>int(_5);               |
++---------------------------------+-----------------+-----------------------------------------------------------------------+
+| logical operator                | AND OR          | select count(*) from stdin where int(1)>123 and int(_5)<200;          |
++---------------------------------+-----------------+-----------------------------------------------------------------------+
+| casting operator                | int(expression) | select int(_1),int( 1.2 + 3.4) from stdin;                            |
++---------------------------------+-----------------+-----------------------------------------------------------------------+
+|                                 |float(expression)|                                                                       |
++---------------------------------+-----------------+-----------------------------------------------------------------------+
+|                                 | timestamp(...)  | select timestamp("1999:10:10-12:23:44") from stdin;                   |
```

Comment:
please give detailed on the supported string formats

Reply:
this table describe what features are currently supported, not explaining their interfaces.

detailed explanations exist on AWS,  or to add a section for more details per function.